contributors/design-proposals/api-machinery/admission-webhook-GA-proposal.md

```diff
@@ -0,0 +1,488 @@
+# Graduate Admission Webhooks to GA
```

Comment:
when you migrate to the enhancements repo, fill in the [KEP template](https://github.com/kubernetes/enhancements/blob/master/keps/0000-kep-template.md) to make sure we're addressing all the required topics